packages/js/src/utils/package-json/update-package-json.ts

```diff
@@ -88,10 +88,17 @@ export function getUpdatedPackageJsonContent(
     options.projectRoot
   );
   const typingsFile = `${relativeMainFileDir}${mainFile}.d.ts`;
-  const exports = {
-    '.': {},
-    ...packageJson.exports,
-  };
+
+  let exports;
+  const { exports: originalExports } = packageJson;
+  exports = originalExports;
+  if (typeof originalExports === 'string') {
+  } else {
+    exports = {
+      '.': {},
+      ...originalExports,
+    };
+  }
```

Comment:
This seems a bit overly complex, I'd either do something like the suggestion below, or at least change the if statement to check if its an object instead of having an empty if block.

```suggestion
  const exports = typeof packageJson.exports === 'string' 
     ? packageJson.exports
     : {
       '.': {}.
       ...packageJson.exports
      };
```